src/styles/globals.css

```
@@ -0,0 +1,75 @@
+ * so shadcn components match the existing app, not the broader PftP CI.
+ *
+ * Reference (hex -> token):
+ *   #68B030 webapp --primary-color (warm green) -> --primary
+ *   #007A49 webapp --primary-dark-color         -> --primary hover / green text on light
+ *   #F0FAF4 soft green                          -> --accent / --secondary
+ *   #F0F6F5 background base                     -> --muted
+ *   #333333 core text                           -> --foreground
+ *   #828282 soft text                           -> --muted-foreground
+ *   #EB5757 fire red                            -> --destructive
+ */
+@tailwind base;
```

Stylelint (17.14.1)

[error] Unexpected unknown at-rule "`@tailwind`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)